distribution/kubeflow/katib/patches/remove-mysql-resources.yaml

```diff
@@ -0,0 +1,39 @@
+#this will remove all resources related to on-cluster MySQL
+---
+$patch: delete
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: katib-mysql-secrets
+  namespace: kubeflow
+---
+# $patch: delete
```

Comment:
I think these will be necessary if `katib-with-kubeflow` is used as a base. Otherwise, there are other resources and customizations that will need to be applied here/

Reply:
That does explain a few things, I was quite confused about why does resources where not being found